### PR TITLE
update graphql default url, and update schema

### PR DESCRIPTION
**Pull request by SpiralP**

### CODE REVIEW

File: src/sdk.ts [hunk 1]
```diff
@@ -53,8 +53,10 @@ export type CloudFlavor = {
 /** Saved Connection info for a Stardog instance */
 export type Connection = {
   __typename?: 'Connection';
+  dashboard?: Maybe<Scalars['String']>;
```
Comment (rmyers) [inline src/sdk.ts:56]: These were the new fields I talked about, we can expose them in the sdk if you add `dashboard` and `index` to the query:

https://github.com/stardog-union/portal-sdk/blob/main/src/documents/listConnectionsQuery.graphql#L2

**Review by rmyers — APPROVED** [review]
Looks good, just a minor tweak to expose the new fields.

File: src/sdk.ts (hunk 1 above)
Comment (SpiralP) [reply src/sdk.ts:62]: ah you have to edit these manually, thanks!

Comment (SpiralP) [reply src/sdk.ts:62]: [`4ab3772` (#3)](https://github.com/stardog-union/portal-sdk/pull/3/commits/4ab377212eaaa92c2a50ea2ef0b2f883506fd118)

**Pull request merged**